concepts/pointers/about.md

```diff
@@ -88,8 +90,8 @@ int main() {
 
 The previous example shows why array indexing begins at `0` instead of `1`.
 The index into an array multiplies the index by the size of the type and adds it to the starting address of the array.
-Since the first element aligns with the beginning of the array, we multiply the type by zero for adding to the start address to get the first element.
-To get the second element we multiple the type by 1 for adding to the start address.
+Since the first element aligns with the beginning of the array, we multiply the type by `0` for adding to the start address to get the first element.
+To get the second element we multiple the type by `1` for adding to the start address.
```

Comment:
Apologies, I missed this first time around.

```suggestion
To get the second element we multiply the type by `1` for adding to the start address.
```

Reply:
I've looked at that so many times and have missed it every time!